plugins/com.google.cloud.tools.eclipse.test.logback/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,12 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.test.logback
```

Comment:
Since this bundle is for test-only purposes, it doesn't much matter.  But `singleton` isn't out of place since we don't want to have multiple `logback.xml`s.